terraform/api_gateway.tf.json.template.py

```diff
@@ -157,6 +157,16 @@ def for_domain(cls, domain):
                     config.blocked_v4_ips_term,
                     config.allowed_v4_ips_term
                 ]
+            },
+            'aws_wafv2_regex_pattern_set': {
+                name: {
+                    'name': config.qualified_resource_name(resource_name=name,
+                                                           stage=config.main_deployment_stage),
+                    'scope': 'REGIONAL',
+                }
+                for name in [
```

Comment:
Don't try to anticipate multiplicity. If the need arises, it often expresses itself in surprising ways, so trying to anticipate it doesn't actually save any time but it comes at the cost of complexity that's not needed today.